DESY/VBF_M125_CP_mt_v1.txt

```diff
@@ -0,0 +1,25 @@
+
+=========
+   v1
+=========
+
+* Year: 2017
+
+* Base sample:
+/VBFHToTauTau_M125_13TeV_powheg_pythia8/RunIIFall17MiniAODv2-PU2017_12Apr2018_new_pmx_94X_mc2017_realistic_v14-v1/MINIAODSIM
+
+* Path to Synchronisation Ntuples: 
+/afs/desy.de/user/f/filatovo/public/CP_Synch_2017/VBF_HToTauTau_M125_CP_mt_v1.root
+
```

Comment:
I am not sure if I have rights to access afs at DESY - I will check it, but generally it is better to store at CERN.

Reply:
Sure, will move to a CERN public directory